lib/sentry/config.ex

```diff
@@ -58,6 +58,13 @@ defmodule Sentry.Config do
               Whether to enable the Oban integration. When enabled, the Sentry SDK will
               capture check-ins for Oban jobs. *Available since v10.2.0*.
               """
+            ],
+            monitor_name_generator: [
+              type: {:fun, 1},
```

Comment:
I think it'd be safer to have the type of this be `{mod, fun}` because captures in config are not generally a good idea (config is used for compilation and yada yada).

Reply:
I just want to double check: 

Do you mean `:mfa` from here? https://hexdocs.pm/nimble_options/NimbleOptions.html#module-types